java/pom.xml

```diff
@@ -29,6 +29,41 @@
     <tag>HEAD</tag>
   </scm>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin</artifactId>
+        <version>6.4.0</version>
```

Comment:
If applied to all project, version management can be done through the `cucumber-parent` pom.